### PR TITLE
docs: migrate documentaion

### PR DESCRIPTION
**Pull request by rishabh3112**
**What kind of change does this PR introduce?**

docs

**Did you add tests for your changes?**
no
**If relevant, did you update the documentation?**
yes

**Does this PR introduce a breaking change?**

No
**Other information**
Refers #247

### CODE REVIEW

File: MIGRATE.md [hunk 1]
```diff
@@ -1,15 +1,29 @@
-## webpack-migrate
-
-The `migrate` feature eases the transition from [version 1](http://webpack.github.io/docs/) to [version 2](https://gist.github.com/sokra/27b24881210b56bbaff7). `migrate`
-also allows users to switch to the new version of webpack without having to extensively [refactor](https://webpack.js.org/guides/migrating/).
-
-### Usage
-To use `migrate`, run the following command, with the value of `<config>` being a path to an existing webpack configuration file
+# webpack-cli migrate
+
+The `webpack-cli migrate` feature eases the transition from version 1 to version 2. `webpack-cli migrate`
```
Comment (ematipico) [inline MIGRATE.md]: And from 2 to 4 (version 3 doesn't have breaking changes in terms of configuration)

File: MIGRATE.md [hunk 2]
```diff
@@ -1,15 +1,29 @@
...
+also allows users to switch to the new version of webpack without having to extensively.
```
Comment (ematipico) [inline MIGRATE.md]: `having to extensively`...? Must complete the phrase. Maybe you meant `having to do it extensively`?

File: MIGRATE.md [hunk 3]
```diff
@@ -1,15 +1,29 @@
...
+### B. Global Setup
+```shell
+$ npm install -g @webpack-cli/migrate
+```
 
+## Usage
+To use `webpack-cli migrate`, run the following command, with the value of `<config>` being a path to an existing webpack configuration file
+### A. Local Setup
+```bash
+npx webpack-cli migrate <config>
```
Comment (ematipico) [inline MIGRATE.md]: I'd add also a real example
```bash
npx webpacli-cli migrate webpack.config.js
```

File: MIGRATE.md [hunk 4]
```diff
@@ -150,12 +164,12 @@ module.exports = {
 
 };
 ```
-
+## Changes Reflected after migration
 In summary, we can see the follow changes were made
 1.  The webpack schema for using loaders has changed
     - `loaders` is now `module.rules`
     -  `query` is now `options`
```
Comment (ematipico) [inline MIGRATE.md]: There are now also other changes (from 2 to 4)
- https://github.com/webpack/webpack-cli/issues/394
- https://github.com/webpack/webpack-cli/issues/395
- https://github.com/webpack/webpack-cli/issues/396

File: MIGRATE.md (hunk 3 above)
Comment (rishabh3112) [reply MIGRATE.md]: See if it's now fine @ematipico after fd7bd9f.

File: MIGRATE.md (hunk 2 above)
Comment (rishabh3112) [reply MIGRATE.md]: My bad 😅.

**Comment by rishabh3112** [comment]
Any changes @ematipico @ev1stensberg ??

**Comment by ematipico** [comment]
Looks good to me! Thank your very much!

**Pull request merged**